include/drivers/clock_control/stm32_clock_control.h

```diff
@@ -75,18 +75,37 @@
 #define STM32_CPU2_PRESCALER	CONFIG_CLOCK_STM32_CPU2_PRESCALER
 #endif
 
+#if DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(rcc), st_stm32h7_rcc, okay) && \
+	DT_NODE_HAS_PROP(DT_NODELABEL(rcc), clocks)
+#define STM32_D1CPRE	DT_PROP(DT_NODELABEL(rcc), d1cpre)
+#define STM32_HPRE	DT_PROP(DT_NODELABEL(rcc), hpre)
+#define STM32_D2PPRE1	DT_PROP(DT_NODELABEL(rcc), d2ppre1)
+#define STM32_D2PPRE2	DT_PROP(DT_NODELABEL(rcc), d2ppre2)
+#define STM32_D1PPRE	DT_PROP(DT_NODELABEL(rcc), d1ppre)
+#define STM32_D3PPRE	DT_PROP(DT_NODELABEL(rcc), d3ppre)
+#else
+#define STM32_D1CPRE	CONFIG_CLOCK_STM32_D1CPRE
+#define STM32_HPRE	CONFIG_CLOCK_STM32_HPRE
+#define STM32_D2PPRE1	CONFIG_CLOCK_STM32_D2PPRE1
+#define STM32_D2PPRE2	CONFIG_CLOCK_STM32_D2PPRE2
+#define STM32_D1PPRE	CONFIG_CLOCK_STM32_D1PPRE
+#define STM32_D3PPRE	CONFIG_CLOCK_STM32_D3PPRE
+#endif
+
 #if DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(pll), st_stm32f2_pll_clock, okay) || \
 	DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(pll), st_stm32f4_pll_clock, okay) || \
 	DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(pll), st_stm32f7_pll_clock, okay) || \
 	DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(pll), st_stm32g0_pll_clock, okay) || \
 	DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(pll), st_stm32g4_pll_clock, okay) || \
 	DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(pll), st_stm32l4_pll_clock, okay) || \
-	DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(pll), st_stm32wb_pll_clock, okay)
+	DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(pll), st_stm32wb_pll_clock, okay) || \
+	DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(pll), st_stm32h7_pll_clock, okay)
 #define STM32_PLL_M_DIVISOR	DT_PROP(DT_NODELABEL(pll), div_m)
 #define STM32_PLL_N_MULTIPLIER	DT_PROP(DT_NODELABEL(pll), mul_n)
 #define STM32_PLL_P_DIVISOR	DT_PROP(DT_NODELABEL(pll), div_p)
 #define STM32_PLL_Q_DIVISOR	DT_PROP(DT_NODELABEL(pll), div_q)
 #define STM32_PLL_R_DIVISOR	DT_PROP(DT_NODELABEL(pll), div_r)
+#warning "ho ho"
```

Comment:
Remove warning

Reply:
Don't like it :-)?

Reply:
Christmas is now far away :-) .
I like comment if message is more explicit.